src/components/Quote.vue

```diff
@@ -247,7 +247,7 @@ export default {
 	padding: 6px 6px 6px 24px;
 	display: flex;
 	max-width: $messages-text-max-width;
-	border-radius: var(--border-radius-large);
+	border-radius: var(--border-radius-element);
```

Comment:
you are still using `--border-radius-large` in `"new-message-form__quote"`

Reply:
![image](https://github.com/user-attachments/assets/147c3963-fa14-4c0c-bb75-c4376953201c) => ![image](https://github.com/user-attachments/assets/869372f9-af1f-499b-bd58-41a3fd86c125)